tests/baselines/reference/user/formik.log

```diff
@@ -0,0 +1,17 @@
+Exit Code: 1
+Standard output:
+index.tsx(30,17): error TS7006: Parameter 'values' implicitly has an 'any' type.
+index.tsx(43,9): error TS7006: Parameter 'values' implicitly has an 'any' type.
+index.tsx(44,11): error TS7031: Binding element 'setSubmitting' implicitly has an 'any' type.
+index.tsx(44,26): error TS7031: Binding element 'setErrors' implicitly has an 'any' type.
+index.tsx(60,9): error TS7031: Binding element 'values' implicitly has an 'any' type.
+index.tsx(61,9): error TS7031: Binding element 'errors' implicitly has an 'any' type.
+index.tsx(62,9): error TS7031: Binding element 'touched' implicitly has an 'any' type.
+index.tsx(63,9): error TS7031: Binding element 'handleChange' implicitly has an 'any' type.
+index.tsx(64,9): error TS7031: Binding element 'handleBlur' implicitly has an 'any' type.
+index.tsx(65,9): error TS7031: Binding element 'handleSubmit' implicitly has an 'any' type.
+index.tsx(66,9): error TS7031: Binding element 'isSubmitting' implicitly has an 'any' type.
```

Comment:
Does this require a fix on our side? Or just a change to react? @weswigham

Reply:
#26281 is half a fix for it (the intersection on the inside). There's also the `string extends keyof P` in the type that won't simplify, though; that'll need to be remodeled so it can somehow resolve even when `P` is still generic (or just be removed - passing `string` into all the `Pick` and friends calls is probably fine).